components/combobox/combobox.jsx

```diff
@@ -407,14 +418,16 @@ class Combobox extends React.Component {
 
 	getNewActiveOptionIndex = ({ activeOptionIndex, offset, options }) => {
 		// used by menu listbox and selected options listbox
-		const nextIndex = activeOptionIndex + offset;
-		const skipIndex =
+		let nextIndex = activeOptionIndex + offset;
+		while (
```

Comment:
Does this enable two separators in a row? What's the use case?

Reply:
Yes, it's more robust. Please replace with a `reduce` call though.

Reply:
Reverting is fine, too, though. No one should be putting two separators next to each other.

Reply:
Ended up reverting it back since it's not related to any changes in this PR.